frontend/components/atoms/animations/scroll.css

```diff
@@ -0,0 +1,44 @@
+/* purgecss start ignore */
```

Comment:
I'm not sure if we should keep scrolling animations by default in the Lean Theme base, what do you think @adamf321 , @Fcog ?

Reply:
I'll keep this for the moment